etc/ci/buildbot_steps.yml

```diff
@@ -11,26 +11,26 @@ mac-rel-wpt1:
 
 mac-rel-wpt2:
   - ./mach clean-nightlies --keep 3 --force
-  - ./mach build --release
+  - ./mach env PKG_CONFIG_PATH=/usr/local/opt/zlib/lib/pkgconfig build --release
```

Comment:
I think env needs to go before mach here, i.e. `env PKG_CONFIG_PATH=... ./mach build --release`